app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -1,69 +1,81 @@
-  private CreatorDashboardAdapter adapter;
 
-  protected @Bind(R.id.creator_dashboard_recycler_view) RecyclerView creatorDashboardRecyclerView;
+  private CreatorDashboardBottomSheetAdapter bottomSheetAdapter;
+  private BottomSheetBehavior bottomSheetBehavior;
+
+  protected @Bind(R.id.creator_dashboard_bottom_sheet_recycler_view) RecyclerView bottomSheetRecyclerView;
 
   @Override
   protected void onCreate(final @Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.creator_dashboard_layout);
     ButterKnife.bind(this);
 
-    this.adapter = new CreatorDashboardAdapter();
-    this.creatorDashboardRecyclerView.setAdapter(this.adapter);
-    this.creatorDashboardRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+    // Set up the bottom sheet recycler view.
+    this.bottomSheetAdapter = new CreatorDashboardBottomSheetAdapter();
+    this.bottomSheetRecyclerView.setAdapter(this.bottomSheetAdapter);
+    this.bottomSheetRecyclerView.setLayoutManager(new LinearLayoutManager(this)); // todo: reuse LayoutManager?
+    this.bottomSheetBehavior = BottomSheetBehavior.from(this.bottomSheetRecyclerView);
 
     this.viewModel.outputs.projectAndStats()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(this::renderProjectAndStats);
+      .subscribe(this::createFragments);
 
-    this.viewModel.outputs.startProjectActivity()
+    this.viewModel.outputs.projectsForBottomSheet()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(projectAndRefTag -> this.startProjectActivity(projectAndRefTag.first, projectAndRefTag.second));
+      .subscribe(this::setProjectsForDropdown);
+  }
+
+  private void setProjectsForDropdown(final List<Project> projects) {
+    this.bottomSheetAdapter.takeProjects(projects);
   }
 
-  private void startProjectActivity(final @NonNull Project project, final @NonNull RefTag refTag) {
-    final Intent intent = new Intent(this, ProjectActivity.class)
-      .putExtra(IntentKey.PROJECT, project)
-      .putExtra(IntentKey.REF_TAG, refTag);
-    startActivity(intent);
+  private void createFragments(final Pair<Project, ProjectStatsEnvelope> projectAndStats) {
+    final FragmentManager fragmentManager = getSupportFragmentManager();
+    final FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+    final CreatorDashboardFragment fragment = CreatorDashboardFragment.newInstance(projectAndStats);
+    fragmentTransaction.add(R.id.creator_dashboard_coordinator_view, fragment);
+    fragmentTransaction.commit();
   }
 
-  private void renderProjectAndStats(final @NonNull Pair<Project, ProjectStatsEnvelope> projectAndStats) {
-    this.adapter.takeProjectAndStats(projectAndStats);
+  public void toggleBottomSheetClick() {
+    this.bottomSheetBehavior.setState(BottomSheetBehavior.STATE_EXPANDED);
+    this.bottomSheetRecyclerView.bringToFront();
```

Comment:
seem chill? just found this command on _internet_ 😎

Reply:
chill!!